File: acmp.ru/001...100/068/68.cpp
#include <iostream>
#include <cstdio>
#include <string>

using namespace std;

int main()
{
	freopen("input.txt","r",stdin);
	freopen("output.txt","w",stdout);
	string s;
	int n;
	cin>>s>>n;
	if (s=="School" && n%2==0) cout<<"NO";
	else cout<<"YES";
	return 0;
}

File: acmp.ru/001...100/068/68.pas
var s:string; n:longint; 
begin 
assign(input, 'input.txt'); reset(input); 
assign(output, 'output.txt'); rewrite(output); 
readln(s); 
readln(n); 
if (s='School') and (n mod 2=0) then writeln('No'); 
if (s='School') and (n mod 2=1) then writeln('Yes'); 
if (s='Home') and (n mod 2=0) then writeln('Yes'); 
if (s='Home') and (n mod 2=1) then writeln('Yes'); 
end.